optimum/onnx/transformations_utils.py

```diff
@@ -312,3 +316,223 @@ def cast_int64_tensorproto_to_int32(initializer: onnx.TensorProto, cast: bool =
 
     initializer.CopyFrom(tensor)
     initializer.name = original_name
+
+
+def _get_weights_to_tie(tied_params, torch_model: "nn.Module") -> Tuple[List[List[str]]]:
+    """
+    Find tied weights in the PyTorch model `model`, and separate them in tied weights
+    for which an untying strategy exists and do not exist.
+    """
+    SUPPORTED_DEDUPLICATION_OPS = ("Embedding", "Linear")
+    tied_groups_to_tie = []
+    tied_groups_ignored = []
+    for params in tied_params:
+        skip_group = False
+        for param_name in params:
+            module_name = ".".join(param_name.split(".")[:-1])
```

Comment:
```suggestion
            module_name = param_name.rsplit(".", maxsplit=1)[0]
```
Not sure it is more clear though